src/views/view_game.py

```diff
@@ -74,17 +74,17 @@ def init_game_btns(self):
         )
 
         disp = (
-            ("card", "Carte", (100, 610)),
-            ("bet", "Bet", (300, 610)),
-            ("end_turn", "End Turn", (500, 610)),
-            ("split", "Split", (700, 610)),
-            ("double", "Double", (900, 610)),
-            ("quit", "Quit", (1100, 610))
+            ("card", "Carte", (40, 610)),
+            ("bet", "Bet", (240, 610)),
+            ("end_turn", "End Turn", (440, 610)),
+            ("split", "Split", (640, 610)),
+            ("double", "Double", (840, 610)),
+            ("quit", "Quit", (1040, 610))
```

Comment:
This should be in a config file

Reply:
I'm waiting to finish my 2 PR to set this in the config files